[Flake]: Race condition building ccache?

Unclear what the problem is. Looks to me like the ccache configuration is not ready or not created but maybe that is a race on the creation order?

https://source.cloud.google.com/results/invocations/4bdd9946-b6c0-41c1-b82f-53a4f0e8d446/targets

```
[ 57%] Building CXX object google/cloud/bigtable/CMakeFiles/bigtable_protos.dir/google/bigtable/admin/v2/instance.pb.cc.o
ccache: error: //.ccache/ccache.conf: No such file or directory
make[3]: *** [google/cloud/bigtable/CMakeFiles/bigtable_protos.dir/google/bigtable/admin/v2/bigtable_instance_admin.pb.cc.o] Error 1
make[3]: *** Waiting for unfinished jobs....
google/cloud/bigtable/CMakeFiles/bigtable_protos.dir/build.make:218: recipe for target 'google/cloud/bigtable/CMakeFiles/bigtable_protos.dir/google/bigtable/admin/v2/bigtable_instance_admin.pb.cc.o' failed
make[2]: *** [google/cloud/bigtable/CMakeFiles/bigtable_protos.dir/all] Error 2
make[1]: *** [CMakeFiles/google-cloud-cpp-dependencies.dir/rule] Error 2
CMakeFiles/Makefile2:3587: recipe for target 'google/cloud/bigtable/CMakeFiles/bigtable_protos.dir/all' failed
CMakeFiles/Makefile2:650: recipe for target 'CMakeFiles/google-cloud-cpp-dependencies.dir/rule' failed
make: *** [google-cloud-cpp-dependencies] Error 2
Makefile:409: recipe for target 'google-cloud-cpp-dependencies' failed
Build finished with 2 exit status Wed Mar  6 15:05:47 PST 2019.
================================================================
```

Here is another one, still unclear what is going on:

https://source.cloud.google.com/results/invocations/882675c9-2291-4706-9c84-960387d667b7/targets/cloud-cpp%2Fgithub%2Fdocker%2Fcheck-abi-presubmit/log

```
[ 56%] Building CXX object CMakeFiles/googleapis_cpp_bigtable_protos.dir/google/bigtable/admin/v2/instance.pb.cc.o
ccache: error: //.ccache/ccache.conf: No such file or directory
CMakeFiles/googleapis_cpp_bigtable_protos.dir/build.make:242: recipe for target 'CMakeFiles/googleapis_cpp_bigtable_protos.dir/google/bigtable/admin/v2/bigtable_table_admin.pb.cc.o' failed
make[6]: *** [CMakeFiles/googleapis_cpp_bigtable_protos.dir/google/bigtable/admin/v2/bigtable_table_admin.pb.cc.o] Error 1
make[6]: *** Waiting for unfinished jobs....
make[5]: *** [CMakeFiles/googleapis_cpp_bigtable_protos.dir/all] Error 2
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/googleapis_cpp_bigtable_protos.dir/all' failed
Makefile:129: recipe for target 'all' failed
make[4]: *** [all] Error 2
google/cloud/bigtable/CMakeFiles/googleapis_project.dir/build.make:112: recipe for target 'external/googleapis/src/googleapis_project-stamp/googleapis_project-build' failed
make[3]: *** [external/googleapis/src/googleapis_project-stamp/googleapis_project-build] Error 2
make[2]: *** [google/cloud/bigtable/CMakeFiles/googleapis_project.dir/all] Error 2
CMakeFiles/Makefile2:4641: recipe for target 'google/cloud/bigtable/CMakeFiles/googleapis_project.dir/all' failed
CMakeFiles/Makefile2:650: recipe for target 'CMakeFiles/google-cloud-cpp-dependencies.dir/rule' failed
make[1]: *** [CMakeFiles/google-cloud-cpp-dependencies.dir/rule] Error 2
make: *** [google-cloud-cpp-dependencies] Error 2
Makefile:409: recipe for target 'google-cloud-cpp-dependencies' failed
Build finished with 2 exit status Wed Mar 27 07:31:32 PDT 2019.
================================================================
================================================================
```

--- a/ci/kokoro/docker/build-in-docker-cmake.sh
+++ b/ci/kokoro/docker/build-in-docker-cmake.sh
@@ -83,6 +83,7 @@ fi
 # shellcheck disable=SC2086
 ${CMAKE_COMMAND} \
     -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" \
+    "-DGOOGLE_CLOUD_CPP_ENABLE_CCACHE=OFF" \
     "${cmake_extra_flags[@]+"${cmake_extra_flags[@]}"}" \
     ${CMAKE_FLAGS:-} \
     "-H${SOURCE_DIR}" \

